routers/api/v1/api.go

```diff
@@ -21,7 +21,7 @@
 //     - text/html
 //
 //     Security:
-//     - BasicAuth :
+//     - BasicOrReverseProxyAuth :
```

Comment:
breaking change ... you can extend but dont remove

Reply:
> you can extend but dont remove

BasicOrReverseProxyAuth is extension of BasicAuth :) basic auth should work without changes after this mod; feel free to fine tune this PR if you see that something can be done more compatible with existing API.

Reply:
well i would add this auth and not replace the old one in the swagger docs so it wont be a breaking change :)

Reply:
Just added separate auth method in api.go comments (code left as it was - feel free to enhance).